_includes/compose/compose-v1-eol.md

```diff
@@ -0,0 +1,7 @@
+> Important
+> 
+> The Python version of Docker Compose is reaching end-of-life.  
+> From April 2023 Compose v1 won't be supported anymore and will be removed from all Docker Desktop versions.  
+> You can already use the Compose v2 with the `docker compose` CLI plugin or by activating the `Use Docker Compose V2` preference in Docker Desktop.   
```

Comment:
Is it worth adding a link to https://docs.docker.com/compose/compose-v2/  To provide a bit more context?

Reply:
That was just a draft, to prepare the communication, we can completely change the wording if needed no worries